content/en/docs/components/katib/experiment.md

```diff
@@ -76,16 +76,17 @@ These are the fields in the experiment configuration spec:
   [`ObjectiveSpec` type](https://github.com/kubeflow/katib/blob/master/pkg/apis/controller/common/v1beta1/common_types.go#L93).
 
 - **parallelTrialCount**: The maximum number of hyperparameter sets that Katib
-  should train in parallel.
+  should train in parallel. The default value is 3.
 
 - **maxTrialCount**: The maximum number of trials to run.
   This is equivalent to the number of hyperparameter sets that Katib should
-  generate to test the model.
+  generate to test the model. If the `maxTrialCount` value is **omitted**, your
+  experiment is running until the objective goal is reached or the experiment
```

Comment:
Can you check if it's "experiment will be running until" instead of "is running"?

```suggestion
  experiment will be running until the objective goal is reached or the experiment
```